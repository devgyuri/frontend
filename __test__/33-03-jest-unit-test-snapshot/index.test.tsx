import JestUnitTestSnapshotPage from "../../pages/section33/33-03-jest-unit-test-snapshot";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";

it("UI test", () => {
  const result = render(<JestUnitTestSnapshotPage />);

  expect(result.container).toMatchSnapshot();
});
