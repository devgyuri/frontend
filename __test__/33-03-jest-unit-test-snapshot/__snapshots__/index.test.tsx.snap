// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI test 1`] = `
<div>
  <div>
    kitty is 5 years old.
  </div>
  hobby: 
  <input
    type="text"
  />
  <button>
    play with kitty
  </button>
</div>
`;
